<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contactos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 222, 222);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAgregar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69, 198, 88);
color: white;
font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 118, 255);
color: white;
font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 56, 59);
color: white;
font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 127, 211);
color: white;
font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbContactos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>321</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLineEdit" name="txtID">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font-size: 13px;
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font-size: 13px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font-size: 13px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>312</width>
      <height>183</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 174, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
